packages/fiori/src/Timeline.ts

```diff
@@ -111,6 +188,116 @@ class Timeline extends UI5Element {
 			: Timeline.i18nBundle.getText(TIMELINE_ARIA_LABEL);
 	}
 
+	get growsOnScroll(): boolean {
+		return this.growing === TimeLineGrowingMode.Scroll;
+	}
+
+	get timeLineEndDOM(): Element {
```

Comment:
For this type of DOM lookups you can make use of the query decorator - it will internally do the same as the getter but shorter as code.


```ts
import query from "@ui5/webcomponents-base/dist/decorators/query.js";

class Timeline {

  @query(".ui5-time-line-end-marker")
  timeLineEndDOM!: HTMLElement;
```